public/app/features/dashboard-scene/scene/layout-default/DashboardGridItem.tsx

```diff
@@ -57,17 +57,31 @@ export class DashboardGridItem
   public constructor(state: DashboardGridItemState) {
     super(state);
 
+    this.addActivationHandler(() => this._handleGridSizeUnsubscribe());
```

Comment:
why a separate activation handler for this?

Reply:
Added it while testing and forgot to merge them into one activation handler >.<. Fixed now.